specs/lib/updateAccessorComponentTypeSpec.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+var Cesium = require('cesium');
+var updateAccessorComponentTypes = require('../../lib/updateAccessorComponentTypes');
+
+var WebGLConstants = Cesium.WebGLConstants;
+
+var source = Buffer.from((new Float32Array([-2.0, 1.0, 0.0, 1.0, 2.0, 3.0])).buffer);
+
+describe('updateAccessorComponentTypes', function() {
+    it('converts joints accessor types', function() {
+        var gltf = {
+            meshes: [
+                {
+                    primitives: [
+                        {
+                            attributes: {
+                                JOINTS_0: 0
+                            }
+                        }, {
+                            attributes: {
+                                JOINTS_0: 1
+                            }
+                        }, {
+                            attributes: {
+                                JOINTS_0: 2
+                            }
+                        }
+                    ]
+                }
+            ],
+            accessors: [
+                {
+                    bufferView: 0,
+                    componentType: WebGLConstants.BYTE,
+                    count: 24,
+                    type: 'VEC4'
+                }, {
+                    bufferView: 1,
+                    componentType: WebGLConstants.FLOAT,
+                    count: 24,
+                    type: 'VEC4'
+                }, {
+                    bufferView: 2,
+                    componentType: WebGLConstants.UNSIGNED_SHORT,
+                    count: 24,
+                    type: 'VEC4'
+                }
+            ],
+            bufferViews: [
+                {
+                    buffer: 0,
+                    byteLength: 12
+                }, {
+                    buffer: 0,
+                    byteOffset: 12,
+                    byteLength: 12
+                }, {
+                    buffer: 0,
+                    byteLength: 12
+                }
+            ],
+            buffers: [
+                {
+                    byteLength: source.byteLength,
+                    extras: {
+                        _pipeline: {
+                            source: source
```

Comment:
We moved all the tests away from setting `_pipeline` directly. Use data uris and `readResources` instead. See [readAccessorPackedSpec](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/2.0/specs/lib/readAccessorPackedSpec.js) for an example.